### DISCUSSION TITLE
Importing Playlists

### DISCUSSION THREAD

**Discussion by ALegitCookie** [MissingCore/Music#95] [General]
Hello everyone, I was just wondering if there is any way to import my playlists from itunes? Being able to import a .xml file in the app settings would be a great addition to help people transition over from their existing music app/device.

**Comment by cyanChill** [comment]
Sorry for the late reply. Currently, there's no way of importing lists from other services - I mainly created the import/backup "feature" in the app so users can transfer their lists between the APK & Google Play version of this app. As of now, I have no plans of doing so as this is probably going to be one of the least used features along with not knowing the general file structure and any quirks that may appear.

Though, with the next major update, we drastically improved how tracks can be populated in playlists (add UI to directly search and add tracks to the playlist instead of manually going through each track to add them to a playlist) which should ease the problem a bit.

**Reply by albertgarcesb96** [reply]
So, aren't you paining adding at least .m3u file support by the moment?  😞

**Comment by cyanChill** [comment]
Having dedicated import methods from other sources probably won't get added in, but experimental support for importing & exporting M3U files to create playlists has been added in the [`v2.6.0-rc.1`](https://github.com/MissingCore/Music/releases/tag/v2.6.0-rc.1) release.
